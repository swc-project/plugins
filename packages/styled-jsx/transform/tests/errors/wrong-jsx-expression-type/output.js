import _JSXStyle from "styled-jsx/style";
export default (()=><div >

    <style jsx>

      {[]}

    </style>

  </div>);
