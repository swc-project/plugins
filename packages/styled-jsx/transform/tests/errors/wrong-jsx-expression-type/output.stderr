
  x Expected a template literal, string or identifier inside the JSXExpressionContainer.
  | Read more: https://nextjs.org/docs/messages/invalid-styled-jsx-children
   ,-[input.js:3:5]
 3 | ,-> <style jsx>
 4 | |         {[]}
 5 | `->     </style>
   `----
